### ISSUE TITLE
NameError: name '_LOGGER' is not defined

### ISSUE THREAD

**Issue by rs443** [gcobb321/icloud3#35]
This error is pretty common i my log. I use icloud3 2.1. Home Assistant 108.8 on docker.

Except from the error in the log tracking is working 👍 
The error happens sometimes when my wifes phone is updating. Never on mine.

```
2020-04-25 08:13:45 INFO (SyncWorker_33) [custom_components.icloud3.device_tracker] Wifes(iphone) FamShr update started
2020-04-25 08:13:45 ERROR (SyncWorker_33) [custom_components.icloud3.pyicloud_ic3]  (450)
2020-04-25 08:13:45 ERROR (SyncWorker_33) [custom_components.icloud3.pyicloud_ic3.http] name '_LOGGER' is not defined
Traceback (most recent call last):
  File "/config/custom_components/icloud3/pyicloud_ic3.py", line 193, in _raise_error
    self.session.cookies.clear()
AttributeError: 'PyiCloudSession' object has no attribute 'session'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/config/custom_components/icloud3/pyicloud_ic3.py", line 128, in request
    self._raise_error(response.status_code, response.reason)
  File "/config/custom_components/icloud3/pyicloud_ic3.py", line 199, in _raise_error
    _LOGGER.exception(err)
NameError: name '_LOGGER' is not defined
2020-04-25 08:13:45 WARNING (SyncWorker_33) [custom_components.icloud3.pyicloud_ic3.http] Failed to parse response with JSON mimetype
2020-04-25 08:13:45 INFO (SyncWorker_33) [custom_components.icloud3.pyicloud_ic3] Authenticating as myemail@gmail.com
2020-04-25 08:13:48 INFO (SyncWorker_33) [custom_components.icloud3.pyicloud_ic3] Authentication completed successfully
2020-04-25 08:13:48 INFO (SyncWorker_33) [custom_components.icloud3.device_tracker] Reauthenticated FamShr account myemail@gmail.com for Wifesiphone
2020-04-25 08:13:48 ERROR (SyncWorker_33) [custom_components.icloud3.device_tracker] '<=' not supported between instances of 'int' and 'NoneType'
Traceback (most recent call last):
  File "/config/custom_components/icloud3/device_tracker.py", line 3476, in _determine_interval_retry_after_error
    latitude, longitude, 0, 0)
  File "/config/custom_components/icloud3/device_tracker.py", line 4327, in _setup_base_kwargs
    elif (zone_dist <= self.zone_radius_m.get(zone_name) and
TypeError: '<=' not supported between instances of 'int' and 'NoneType'
2020-04-25 08:13:50 INFO (SyncWorker_35) [custom_components.icloud3.device_tracker] Wifes(iphone) Device Tracking Complete, Results-BaseZone-home > Zone-jobbl, GPS-(59.2162, 15.3451), Interval-20 min, Dist-5.04 km, TravTime- (in_zone), NextUpdt-10:33:45, Located-09:43:40 (-5390 sec ago)
```

icloud3 configuration:
```
device_tracker:
  - platform: icloud3
    username: !secret myname_iphoneuser
    password: !secret myname_iphonepass
    group: familjen
    tracking_method: famshr
    track_devices:
      - mynameiphone > myname.png
      - wifesiphone > wife.png
    gps_accuracy_threshold: 66
    unit_of_measurement: km
    inzone_interval: 20
    stationary_inzone_interval: 15
    stationary_still_time: 20
    distance_method: calc
    create_sensors: trig
```

**Comment by gcobb321** [comment]
The iCloud Location Service program (pyicloud_ic3.py) is returning a 450 error noted on the 2nd line with the `(450)`. This indicates some type of authentication problem and that version of nd the pyicloud3.py program has recently been updated. I am now testing the new version. I am also working on iCloud3 v2.2 which includes an updated version of pyicloud_ic3.py. Hopefully, this problem will be fixed.

I noticed you are using the famshr (Family Share) tracking method. This logs into the normal Apple account with 2fa and will generate message every 1/2 hour or so that a web service has logged into your account. This is described in the iCloud3 documentation [here](https://gcobb321.github.io/icloud3/#/chapters/1-setting-up-icloud-account?id=icloud-and-2-factor-authentication). iC3 v2.2 will also solve this problem. I'll be posting a development version when it is ready sometime in the next 2 weeks.

In the mean time, I would suggest you try the FmF tracking method which authenticates a little differently and may solve the problem. I also noticed an error message dealing with the zone radius . I can't tell if the first error caused the zone radius error or not but would suggest you check the zones you have set up in HA.

I'll post a not here when v2.2 is ready to try.
I would suggest
